src/stores/evaluationStore.js

```diff
@@ -51,6 +53,7 @@ const evaluationContext = {
   WCAG20: 'http://www.w3.org/TR/WCAG20/#',
   WCAG21: 'http://www.w3.org/TR/WCAG21/#',
   wcagVersion: 'WAI:standards-guidelines/wcag/#versions',
+  version: 'wcagem:version',
```

Comment:
misschien willen we dit `reportToolVersion` oid noemen voor extra duidelijkheid?